resources/js/Pages/Vault/Dashboard/Task/Index.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <layout :layout-data="layoutData" :inside-vault="true">
+    <main class="relative sm:mt-24">
+      <div class="mx-auto max-w-3xl px-2 py-2 sm:py-6 sm:px-6 lg:px-8">
+        <!-- title -->
+        <div class="mb-5 items-center justify-between border-b border-gray-200 pb-2 sm:flex">
+          <div class="mb-2 sm:mb-0">
+            <span class="relative">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                class="icon-sidebar relative inline h-4 w-4 text-gray-300 hover:text-gray-600"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor">
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  stroke-width="2"
+                  d="M9 5H7a2 2 0 00-2 2v12a2 2 0 002 2h10a2 2 0 002-2V7a2 2 0 00-2-2h-2M9 5a2 2 0 002 2h2a2 2 0 002-2M9 5a2 2 0 012-2h2a2 2 0 012 2m-6 9l2 2 4-4" />
+              </svg>
+            </span>
+
+            {{ $t('vault.tasks_title') }}
+          </div>
+        </div>
+
+        <div v-for="task in data" :key="task.id" class="mb-6">
+          <!-- person name -->
+          <div class="mb-2 flex items-center">
+            <avatar :data="task.contact.avatar" :classes="'mr-2 h-5 w-5 rounded-full'" />
+
+            <inertia-link :href="task.contact.url.show" class="text-blue-500 hover:underline">
+              {{ task.contact.name }}
+            </inertia-link>
+          </div>
+
+          <!-- tasks -->
+          <ul class="rounded-lg border border-gray-200 bg-white">
+            <li
+              v-for="currentTask in task.tasks"
+              :key="currentTask.id"
+              class="item-list border-b border-gray-200 px-3 py-2 hover:bg-slate-50">
+              <div class="flex items-center">
+                <input
+                  :id="currentTask.id"
+                  v-model="currentTask.completed"
+                  :name="currentTask.id"
+                  type="checkbox"
+                  class="focus:ring-3 relative h-4 w-4 rounded border border-gray-300 bg-gray-50 focus:ring-blue-300 dark:border-gray-600 dark:bg-gray-700 dark:ring-offset-gray-800 dark:focus:ring-blue-600"
+                  @change="toggle(currentTask)" />
+
+                <label :for="currentTask.id" class="ml-2 flex cursor-pointer items-center text-gray-900">
+                  {{ currentTask.label }}
+
+                  <!-- due date -->
+                  <span
+                    v-if="currentTask.due_at"
+                    :class="currentTask.due_at_late ? 'bg-red-400/10 text-red-600' : 'bg-sky-400/10 text-sky-600'"
+                    class="ml-2 flex items-center rounded-full px-2 py-0.5 text-xs font-medium leading-5 text-sky-600 dark:text-sky-400">
```

Comment:
```suggestion
                    :class="currentTask.due_at_late ? 'bg-red-400/10 dark:bg-red-600/10 text-red-600 dark:text-red-400' : 'bg-sky-400/10 dark:bg-sky-600/10 text-sky-600 dark:text-sky-400'"
                    class="ml-2 flex items-center rounded-full px-2 py-0.5 text-xs font-medium leading-5">
```

Reply:
This changes the layout to:
![image](https://user-images.githubusercontent.com/25419741/188145967-2895b92f-f353-4cd7-8364-a5440df8e416.png)
is it what you wanted? if not you'll have to modify it, but you can't have multiple text color definition in the same result.

Reply:
dark version: 
![image](https://user-images.githubusercontent.com/25419741/188146481-838b5d4c-9db1-4f7e-a9b0-fe1dd887de2b.png)

Reply:
I love it.